features/filewriter.feature

```diff
@@ -1,15 +1,9 @@
 Feature: dp-sitemap write robot file
 
     Scenario: Write simple robots file
-        Given i have the following robot.json:
+        Given i have the following robot folder:
```

Comment:
Better to do this as an inline arg. See line 8 for an example.

Reply:
also - why a folder and not a specific file?

Reply:
possibly because you've got multiple files in the folder I guess? 

so maybe 'given I have my robots config files in the "<folderName"> folder

Reply:
Thank you Linden. This is now addressed.